pysoa/test/stub_service.py

```diff
@@ -353,14 +353,15 @@ def call_body(self):
         def call_bodies(self):
             return tuple(args[0][0] for args in self.call_args_list)
 
-    def __init__(self, service, action, body=None, errors=None):
+    def __init__(self, service, action, body=None, errors=None, side_effect=None):
```

Comment:
I think I would like to call this `function` instead of `side_effect`. Given that this is a stub (even if it uses a Mock to do the heavy lifting), I feel like `function` more clearly portrays its purpose. What do you think? I'm not hard set on that.

Reply:
I'd rather stick to `side_effect`:
```
...

Alternatively side_effect can be an exception class or instance. In this case the exception will be raised when the mock is called.

If side_effect is an iterable then each call to the mock will return the next value from the iterable.
```

Reply:
Okay.